### ISSUE TITLE
Missing parameter when call include in refresh method

### ISSUE THREAD

**Issue by L0rD59** [mnot/hinclude#31]
Hi,

in refresh method you call include method, but missing media parameter : 

refresh method : 
```
    refresh: function (element_id) {
      var i = 0;
      var mode = this.get_meta("include_mode", "buffered");
      var callback = function (element, req) {};
      callback = this.set_content_buffered;
      for (i; i < this.includes.length; i += 1) {
        if (this.includes[i].getAttribute("id") === element_id) {
          this.include(this.includes[i], this.includes[i].getAttribute("src"), callback);
        }
      }
    },
```
include method : 
```
include: function (element, url, media, incl_cb) { ... }
```

will be : 
```
    refresh: function (element_id) {
      var i = 0;
      var mode = this.get_meta("include_mode", "buffered");
      var callback = function (element, req) {};
      callback = this.set_content_buffered;
      for (i; i < this.includes.length; i += 1) {
        if (this.includes[i].getAttribute("id") === element_id) {
          this.include(this.includes[i], this.includes[i].getAttribute("src"), this.includes[i].getAttribute("media"), callback);
        }
      }
    },
```

**Comment by mnot** [comment]
Fixed by fb033ad, I think.

**Comment by L0rD59** [comment]
I think too :+1:
